tests/phpunit/tests/rest-api/rest-post-meta-fields.php

```diff
@@ -3095,6 +3107,19 @@ public function test_default_is_added_to_schema() {
 		$this->assertSame( 'Goodnight Moon', $schema['default'] );
 	}
 
+	/**
+	 * @ticket 61998
+	 */
+	public function test_title_is_added_to_schema() {
+		$request  = new WP_REST_Request( 'OPTIONS', '/wp/v2/posts' );
+		$response = rest_do_request( $request );
+
+		$schema = $response->get_data()['schema']['properties']['meta']['properties']['with_label'];
+
+		$this->assertArrayHasKey( 'default', $schema );
+		$this->assertSame( 'Meta Label', $schema['title'] );
```

Comment:
Is `default` correct here or are you intending to test for `title`?

As the test has multiple assertions, it will need to include messages for each to make it easier to discover the cause of failures.

```suggestion
		$this->assertArrayHasKey( 'default', $schema, 'Schema is expected to have the default property' );
		$this->assertSame( 'Meta Label', $schema['title'], 'Schema title is expected to be defined.' );
```

Reply:
It looks like a copy and paste issue from `test_default_is_added_to_schema` test 😄

Reply:
You're totally right. Thanks for pointing that out. I've changed that in this commit: https://github.com/WordPress/wordpress-develop/pull/7298/commits/fcb7b6321dcc4e5c9748df005ad4469c976df8c4

I took the opportunity to include the messages in the other test I used as reference. I hope that's fine.